tests/integration_test/transcoding_errors_test.go

```diff
@@ -60,8 +59,7 @@ func TestTranscodingServiceUnavailableError(t *testing.T) {
 		clientProtocol: "http",
 		httpMethod:     "GET",
 		method:         "/v1/shelves/200/books/2001?key=api-key",
-		noBackend:      true,
-		wantErr:        "503 Service Unavailable",
+		wantErr:        `503 Service Unavailable, {"code":503,"message":"upstream connect error or disconnect/reset before headers. reset reason: connection failure"}`,
```

Comment:
This is expected, the backend is unavailable. But the error in the next test is not.